src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
```

Comment:
- [ ] Great job working on this milestone! 💯 However, please consider removing all commented pieces of code, they make your code look dirty and less readable! 🤓 🔧

Reply:
Tnx for your attention, I uncommented them because it seems they don't make issue